src/box/rt_lua_call_access.h

```diff
@@ -0,0 +1,70 @@
+#pragma once
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <zlib.h>
+#include <lua.h>
+#include <lauxlib.h>
+#include <lualib.h>
+
+#if defined(__cplusplus)
+extern "C" {
+#endif /* defined(__cplusplus) */
+
+#define is_super(u) ((u)->roles.m[0] == 20u)
+#define is_universe(s, l) ((l) == 0 || (s)[0] == '\0')
+#define RT_ACCESS 1
+#define RT_NO_ACCESS 0
+
+const char *
+user_name_by_id(uint32_t uid, uint32_t *name_len);
+
+/**
+ * Initialize the user access cache.
+ */
+void
+rt_lua_call_access_init(void);
+
+/**
+ * Destroy the user access cache.
+ */
+void
+rt_lua_call_access_destroy(void);
+
+/**
+ * Grant a access to lua_call function for a user.
+ * @param uname Name of the user.
+ * @param uname_len Length of the user name.
+ * @param fname Name of the function.
+ * @param fname_len Length of the function name.
+ */
+void
+grant_rt_access(const char *uname, uint32_t uname_len,
+		const char *fname, uint32_t fname_len);
```

Comment:
We generally prepent a function name with its compilation unit name.

Reply:
renamed.